.circleci/config.yml.j2

```diff
@@ -131,13 +131,25 @@ jobs:
           when: always
 {%- endfor %}
 
+  check-commander-airflow-version:
+    docker:
+      # https://hub.docker.com/_/docker?tab=tags
+      - image: cimg/base:2021.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Check that commander image uses same Airflow chart version
+          command: ./bin/validate_commander_airflow_version
+
 workflows:
   version: 2.1
   build-and-release-helm-chart:
     jobs:
       - lint
       - run_pre_commit
       - unittest-charts
+      - check-commander-airflow-version
```

Comment:
Given that this is going to stop the CI workflow if it fails, could this be implemented as a pytest test case? Or could this shell script be included as part of the `make unittest-charts` target?

Reply:
we can adding it to makefile so that QA can also make a quick validation.
Pytest way seems good so all test cases are handled in single place

Reply:
I thought about making it as a test case, but I thought about the following:
- Somebody makes a change to another completely independent part of the chart, let's say FluentD's subchart
- Test run, and they fail because Airflow chart version mismatches in Commander and Astronomer
- Might create confusion for the user, and the reviewers. I believe the reaction should be "well, we'll create another PR to bump the airflow chart version either in Commander or in Astronomer". But the "tests" CI job will still fail until we do, leading to confusion.
- By creating a new job for this specific purpose, tests can still pass if versions mismatch, no confusion for devs + reviewers, we can decide to ignore this fail and create a second PR to bump up version (or do it in Commander).
- Another point that motivated this decision to split it up into a new job, is that we also have contributors from the community, who don't have access to commander's repository, so they can't even make the second PR to fix airflow chart version. So I wanted this check to be independant from the chart tests that the user might update/contribute to.

I'll create a Make command to run this script, to allow QA folks to test it easily. What do you think, do you want me to move it to a Pytest test case anyway ?

Reply:
+1 to add make subcommand and trigger it during `make unittest-charts` via dependency.

Reply:
regarding pytest not sure it will slow down chart testing.

Reply:
I'm Ok to add it to `make unittest-charts` as a dependency, but the "problem" is the same one I've cited in my previous comment -> the CI step `test` will fail, which leads to confusion.

In my opinion, I prefer to leave it as a separate make command (I've pushed `make validate-commander-airflow-version`) so that there's a different CI job/step running ONLY for this purpose, and it doesn't create confusion "like why are the tests failing" ?

Reply:
i think a good middle ground is to have a make target for `make test-commander-version` and have a new target `test: test-commander-version unittest-charts`

Reply:
I added a new Make target `test` that uses `validate-commander-airlfow-version` and `unittest-charts` as described here.